### DISCUSSION TITLE
Move code that converts ddf tensor to itk VectorImage into library?

### DISCUSSION THREAD

**Discussion by HastingsGreer** [Project-MONAI/MONAI#4104] [General]
There is a correct way to convert a ddf tensor accepted by Monai's warp block into an itk VectorImage accepted by itk's WarpImageFilter. 

This conversion is defined and tested here:

[https://github.com/Project-MONAI/MONAI/blob/af6560a5facef14483e43cb516d8f07d72aa1ffe/tests/test_warp.py#L178](https://github.com/Project-MONAI/MONAI/blob/af6560a5facef14483e43cb516d8f07d72aa1ffe/tests/test_warp.py#L178)
)

I would like to use this conversion code, and I think it is generally useful since it correctly handles fiddly details like transposing the tensor and reversing the order of elements in each vector. Would a pull request moving it from the test code into the MONAI library be welcome?

**Comment by wyli** [comment]
I think so, let me convert this into a ticket... cc @kate-sann5100
